### PR TITLE
Add options to rolate the log file

### PR DESCRIPTION
**Pull request by starchow**
I have problem with the log file, my server was out of storage when the log file was too large (6GB).
This pull request will allow user config the limit of log file.
Please review and merge it to the master repository if it's good enough.

Thanks you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +93,9 @@ Where `<environment>` is your Rails environment and `<options>` can be:
     -n, --error-notification         Enables error notifications via Airbrake or Bugsnag.
     -F, --feedback-poll N            Frequency in seconds to check for feedback for the feedback processor. Default: 60. Use 0 to disable.
     -b, --feedback-processor PATH    Path to the feedback processor. Default: lib/push/feedback_processor.
+    -r, --rolate-log                 Rolate the log file to limit the log size.
```
Comment (biinari) [inline README.md]: This should be 'rotate-log' instead of 'rolate-log'

File: README.md [hunk 2]
```diff
@@ -92,6 +93,9 @@ Where `<environment>` is your Rails environment and `<options>` can be:
...
+    -S, --log-size N                 Number of log files. Default: 5.
```
Comment (biinari) [inline README.md]: I wonder if this would be better named as `--log-keep` for how many log files are kept

File: README.md (hunk 1 above)
Comment (tompesman) [inline README.md]: @biinari agree

File: README.md (hunk 2 above)
Comment (tompesman) [inline README.md]: @biinari agree, that's better, `log-size` is confusing

**Pull request closed without merging**